include/xtensor/xaccumulator.hpp

```diff
@@ -25,18 +25,33 @@ namespace xt
 
 #define DEFAULT_STRATEGY_ACCUMULATORS evaluation_strategy::immediate_type
 
+    namespace detail
+    {
+        template <class V = void>
+        struct accumulator_identity: public xtl::identity
+        {
+            using value_type = V;
+
+            template <class T>
+            T&& operator()(T&& x) const
```

Comment:
No need to redefine `operator()` since it is inherited from `xtl::identity`

Reply:
done